src/server/serveractiveobject.h

```diff
@@ -141,6 +141,10 @@ class ServerActiveObject : public ActiveObject
 	virtual u16 getHP() const
 	{ return 0; }
 
+	/// Always returns the same unique string for the same object.
+	/// Because these strings are very short, copying them is not expensive.
```

Comment:
```suggestion
	/// @brief Returns an unique ID for this object (persistent across unload, server restarts).
	/// @note Because these strings are very short, copying them is not expensive.
```